LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/BasicsStationNetworkServerStartup.cs

```diff
@@ -49,13 +50,15 @@ public void ConfigureServices(IServiceCollection services)
                         .AddSingleton<ILoRaDeviceRegistry, LoRaDeviceRegistry>()
                         .AddSingleton<IJoinRequestMessageHandler, JoinRequestMessageHandler>()
                         .AddSingleton<IMessageDispatcher, MessageDispatcher>()
-                        .AddTransient<ILnsProtocolMessageProcessor, LnsProtocolMessageProcessor>()
-                        .AddSingleton<IBasicsStationConfigurationService, BasicsStationConfigurationService>();
+                        .AddSingleton<IBasicsStationConfigurationService, BasicsStationConfigurationService>()
+                        .AddSingleton<IClassCDeviceMessageSender, DefaultClassCDevicesMessageSender>()
```

Comment:
DefaultClassCDevicesMessageSender requires a IPacketForwarder to be injected.
While I am still working on the uplink/downlink support, we currently have not a valid alternative for BasicsStation. My suggestion is to temporarily create a fake implementation for this PR

Reply:
Added fake imple